transform/mattermost-analytics/models/staging/mm_telemetry_prod/_mm_telemetry_prod__models.yml

```diff
@@ -21,4 +21,74 @@ models:
       - name: server_id
         description: The ID of the server the event originated from.
       - name: received_at
-        description: Timestamp registered by RudderStack when the event was ingested (received).
\ No newline at end of file
+        description: Timestamp registered by RudderStack when the event was ingested (received).
+
+    - name: stg_mm_telemetry_prod__performance_events
```

Comment:
why not add some generic tests at least for the "main" columns? (here and for `rc`)

Reply:
I can think of one test, since they use the field `duration` to measure performance, I'll check with the team and add them.